packages/amplify-graphql-schema-generator/src/schema-representation/engine.ts

```diff
@@ -0,0 +1,6 @@
+export type DBEngineType = 'MySQL' | 'DynamoDB';
```

Comment:
Is MySQL the right granulatity here, or do we want 'Relational' vs. 'NoSQL'? or something like that?

Reply:
This value is intended to determine the queries required to retrieve table metadata. The logic differs even between relational databases, so it makes sense to have the actual DB engine names.